src/main/java/mods/railcraft/world/level/levelgen/feature/RailcraftFeatures.java

```diff
@@ -0,0 +1,23 @@
+package mods.railcraft.world.level.levelgen.feature;
+
+import net.minecraft.core.registries.Registries;
+import net.minecraft.world.level.levelgen.feature.Feature;
+import net.neoforged.bus.api.IEventBus;
+import net.neoforged.neoforge.registries.DeferredHolder;
+import net.neoforged.neoforge.registries.DeferredRegister;
+
+import mods.railcraft.api.core.RailcraftConstants;
+import mods.railcraft.world.level.levelgen.feature.configuration.QuarriedConfiguration;
+
+public class RailcraftFeatures {
+
+    private static final DeferredRegister<Feature<?>> FEATURES =
```

Comment:
I know it's right to put `FEATURES`, but better to put `deferredRegister` for consistency with the rest. One day I'll change the name of all of them.
```suggestion
    private static final DeferredRegister<Feature<?>> deferredRegister =
```